mscore/preferences.cpp

```diff
@@ -1149,6 +1151,18 @@ void PreferenceDialog::resetShortcutClicked()
       active->setText(1, shortcut->keysToString());
       shortcutsChanged = true;
       }
+    
+void PreferenceDialog::saveShortcutListClicked()
+      {
+      QString saveFileName = QFileDialog::getSaveFileName(new QWidget(),"Save Shortcuts", tr("XML Files(*.xml)"));
```

Comment:
Where you are back on IRC, ping me, and explain me how you would search the source code for other use of `getSaveFileName`